### PR TITLE
Fix warning in the gsch2pcb netlist backend

### PR DESCRIPTION
**Pull request by graahnul-grom**

### CODE REVIEW

File: netlist/scheme/gnet-gsch2pcb.scm.in [hunk 1]
```diff
@@ -182,8 +184,15 @@
   (let* ((c2p (pipe))  ; child to parent
          (p2c (pipe))) ; parent to child
 
+    ; TODO: make setvbuf( port, mode ) work with both Guile 2.0 and 2.2
```
Comment (vzh) [inline netlist/scheme/gnet-gsch2pcb.scm.in]: Have you thought about using a conditional here? Say, you could check guile `effective-version` or `minor-version` to opt appropriate `mode` value. E.g.
```
(setvbuf (cdr c2p) (if (string= (minor-version) "0") _IONBF 'none))

Comment (graahnul-grom) [reply netlist/scheme/gnet-gsch2pcb.scm.in]: @vzh Thank you. :)

**Comment by graahnul-grom** [comment]
@vzh Unfortunately, just using conditional statement here doesn't help: Guile 2.2 still issues the same warning. My bad, I overlooked that somehow.
I found that `(eval-string "_IONBF")` works.

**Comment by vzh** [comment]
How to trigger it? I don't see any suspicious warnings when using `lepton-netlist -g gsch2pcb file.sch` built with current master.

**Comment by graahnul-grom** [comment]
The warning appear when Guile 2.2 compiles `gsch2pcb.scm` file.

**Pull request merged**